desktop/conf.dist/hue.ini

```diff
@@ -1937,6 +1937,9 @@
   # Signed certificate to send along with encrypted metadata.
   ## cert_file=
 
+  # If your computer and another computer that you are communicating with are not in synch regarding the computer clock, then here you can state how big a difference you are prepared to accept.
```

Comment:
Also add to https://github.com/cloudera/hue/blob/master/desktop/conf/pseudo-distributed.ini.tmpl?

(sorry, we need to keep both in syncs)


Then good to go and nice work!